### PR TITLE
26133: Fixed - Coupon code text field not display in proper width in …

### LINKED ISSUE #26133: Coupon code text field not display in proper width in  Internet Explorer/EDGE browser
Coupon code text field not display in proper width in Internet Explorer/EDGE browser

### Preconditions (*)
Magento : 2.3.3

### Steps to reproduce (*)
Review the coupon code section in IE or Edge browser

### Expected result (*)
It should display same as per chrome/firefox browsers

image

### Actual result (*)
image

### PR DESCRIPTION
**Pull request by solwininfotech**
…Internet Explorer/EDGE browser

### Fixed Issues (if relevant)

1. Fixes magento/magento2#26133  Coupon code text field not display in proper width in Internet Explorer/EDGE browser

### Manual testing scenarios (*)

1. Add a product to the cart
2. Go to Shipping Cart Page

Before Fix

26133_before

After Fix 

26133_after

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by solwininfotech** [comment]
@magento give me test instance

**Comment by solwininfotech** [comment]
@magento give me test instance

**Comment by solwininfotech** [comment]
@magento run all tests

**Comment by vijay-wagento** [comment]
#dmcdindia2020

**Comment by solwininfotech** [comment]
@magento give me test instance

**Comment by solwininfotech** [comment]
@magento run all tests

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less [hunk 1]
```diff
@@ -727,6 +727,13 @@
                     position: static;
                 }
             }
+            .content {
+                .fieldset {
+                    .actions-toolbar {
```
Comment (ptylek) [inline app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less]: `.actions-toolbar` selector receives `width: 1%` from `Magento_Paypal/web/css/source/module/_review.less`. Can you check it more thoroughly?

Please also check if this works with just `.block { .actions-toolbar` selector path (remove .content and .fieldset selectors). Add empty lines before and after styles.

**Review by ptylek — CHANGES_REQUESTED** [review]

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less (hunk 1 above)
Comment (solwininfotech) [reply app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less]: @ptylek, thank you for your revision changes that have been done and pushed.

**Comment by solwininfotech** [comment]
@magento run all tests

**Review by ptylek — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ptylek, thank you for the review.
[ENGCOM-8342](https://jira.corp.magento.com/browse/ENGCOM-8342) has been created to process this Pull Request

**Comment by solwininfotech** [comment]
@ptylek  thank you for your guidance and approving changes.

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

### Manual testing scenario

1. Open Storefront from **Internet Explorer** browser
2. Add any product to Cart
3. Click on Cart, select **View and Edit cart**
4. Click on **Apply Discount Code**
5. Observe the _Enter discount code_ field size

**BEFORE** applying changes provided in this PR
Before fix

**AFTER** switching to the PR
After fix

### Note
Also rechecked on Google Chrome browser, the field is not broken

**Comment by solwininfotech** [comment]
Thank you @ptylek  and all to help me to complete this ticket.

**Pull request merged**